code/game/atoms.dm

```diff
@@ -312,6 +312,8 @@ var/list/blood_splatter_icons = list()
 		blood_DNA = null
 		return 1
 
+/atom/proc/wash_cream()

```

Comment:
just a note but if we get any more of these it's gonna start making sense to have some kind of overlays/datum system to handle it.